cob_moveit_bringup/launch/move_group.launch

```diff
@@ -38,6 +39,7 @@
   <include ns="move_group" file="$(find cob_moveit_bringup)/launch/controller_manager.xml" if="$(arg allow_trajectory_execution)">
     <arg name="config_path" value="$(arg config_path)"/>
     <arg name="fake_execution" value="$(arg fake_execution)"/>
+    <arg name="moveit_controller_manager" value="fake" if="$(arg fake_execution)"/>
```

Comment:
`if` condition goes first...
also, fake_execution is only used in demo.launch...please remove